libs/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_subdirectory(crypto)
 add_subdirectory(datetime)
 add_subdirectory(timer)
-add_subdirectory(dao)
\ No newline at end of file
+add_subdirectory(dao)
+add_subdirectory(logger)
```

Comment:
Should logger be in libs/?
The feature of logger will have not only current std::cout logging but also telling clients the logs. It seems so fat that logger is in libs/.